lib/travis/api/v3/services/insights_spotlight_summary/all.rb

```diff
@@ -0,0 +1,12 @@
+module Travis::API::V3
+    class Services::InsightsSpotlightSummary::All < Service
+      params :time_start, :time_end, :repo_id, :build_status
+      result_type :insights_spotlight_summary
+  
+      def run!
+        raise LoginRequired unless access_control.full_access_or_logged_in?
+        result query(:insights_spotlight_summary).all(access_control.user.id)
+      end
+    end
+  end
+  
```

Comment:
@chinmaychaudhari-accolite pls reformat the code. put nee line a the end of file

Reply:
@vitalied there is new line present at the EOF for this file

Reply:
@chinmaychaudhari-accolite reformat the code